### ISSUE TITLE
Bug when 2 wysiwyg editors are on category edit page or product edit page

### ISSUE THREAD

**Issue by mmatrafailo** [magento/magento2#18534]
Pull request https://github.com/magento/magento2/pull/18535
### Preconditions
2 or more wysiwyg editors on catalog product edit page, category edit page.

### Steps to reproduce:
1. Have 2 or more wysiwyg editors on catalog category edit page;
2. Press "WYSIWYG Editor" popup button on first editor;
3. Fill some text data in opened popup form.
4. Submit popup.
5. Press "WYSIWYG Editor" popup button on second editor;
6. Fill some text data in opened popup form.
7. Submit popup.

### Expected Result:
The filled data isn't changed(reseted) after second editor popup submit.

### Actual Result
The data of first editor have been lost after second editor popup submit.

### category_form_xml
```
<fieldset name="content">
        <field name="description" template="ui/form/field" sortOrder="52" formElement="wysiwyg">
            <argument name="data" xsi:type="array">
                <item name="config" xsi:type="array">
                    <item name="wysiwygConfigData" xsi:type="array">
                        <item name="settings" xsi:type="array">
                            <item name="theme_advanced_buttons1"
                                  xsi:type="string">magentowidget,bold,italic,|,justifyleft,justifycenter,justifyright,|,fontselect,fontsizeselect,|,forecolor,backcolor,|,link,unlink,image,|,bullist,numlist,|,code
                            </item>
                            <item name="theme_advanced_buttons2" xsi:type="boolean">false</item>
                            <item name="theme_advanced_buttons3" xsi:type="boolean">false</item>
                            <item name="theme_advanced_buttons4" xsi:type="boolean">false</item>
                            <item name="theme_advanced_statusbar_location" xsi:type="boolean">false</item>
                        </item>
                        <item name="files_browser_window_url" xsi:type="boolean">false</item>
                        <item name="height" xsi:type="string">150px</item>
                        <item name="toggle_button" xsi:type="boolean">false</item>
                        <item name="add_variables" xsi:type="boolean">false</item>
                        <item name="add_widgets" xsi:type="boolean">true</item>
                        <item name="add_images" xsi:type="boolean">false</item>
                        <item name="add_directives" xsi:type="boolean">true</item>
                    </item>
                    <item name="source" xsi:type="string">category</item>
                </item>
            </argument>
            <settings>
                <label translate="true">Description</label>
                <dataScope>description</dataScope>
            </settings>
            <formElements>
                <wysiwyg class="Magento\Catalog\Ui\Component\Category\Form\Element\Wysiwyg">
                    <settings>
                        <rows>8</rows>
                        <wysiwyg>true</wysiwyg>
                    </settings>
                </wysiwyg>
            </formElements>
        </field>
        <field name="more_content" template="ui/form/field" sortOrder="53" formElement="wysiwyg">
            <argument name="data" xsi:type="array">
                <item name="config" xsi:type="array">
                    <item name="wysiwygConfigData" xsi:type="array">
                        <item name="settings" xsi:type="array">
                            <item name="theme_advanced_buttons1"
                                  xsi:type="string">magentowidget,bold,italic,|,justifyleft,justifycenter,justifyright,|,fontselect,fontsizeselect,|,forecolor,backcolor,|,link,unlink,image,|,bullist,numlist,|,code
                            </item>
                            <item name="theme_advanced_buttons2" xsi:type="boolean">false</item>
                            <item name="theme_advanced_buttons3" xsi:type="boolean">false</item>
                            <item name="theme_advanced_buttons4" xsi:type="boolean">false</item>
                            <item name="theme_advanced_statusbar_location" xsi:type="boolean">false</item>
                        </item>
                        <item name="files_browser_window_url" xsi:type="boolean">false</item>
                        <item name="height" xsi:type="string">150px</item>
                        <item name="toggle_button" xsi:type="boolean">false</item>
                        <item name="add_variables" xsi:type="boolean">false</item>
                        <item name="add_widgets" xsi:type="boolean">true</item>
                        <item name="add_images" xsi:type="boolean">false</item>
                        <item name="add_directives" xsi:type="boolean">true</item>
                    </item>
                    <item name="source" xsi:type="string">category</item>
                </item>
            </argument>
            <settings>
                <label translate="true">Additional content</label>
                <dataScope>more_content</dataScope>
            </settings>
            <formElements>
                <wysiwyg class="Magento\Catalog\Ui\Component\Category\Form\Element\Wysiwyg">
                    <settings>
                        <rows>8</rows>
                        <wysiwyg>true</wysiwyg>
                    </settings>
                </wysiwyg>
            </formElements>
        </field>
    </fieldset>
```
### Attribute setup data

```
$attributeData=
    [
        'label' => 'Additionnal content',
        'code' => 'more_content',
        'type' => 'text',
        'input' => 'textarea',
        'required' => false,
        'sort_order' => 40,
        'global' => \Magento\Eav\Model\Entity\Attribute\ScopedAttributeInterface::SCOPE_STORE,
        'wysiwyg_enabled' => true,
        'is_html_allowed_on_front' => true,
        'group' => 'Content',
    ];
```
### Category Edit Page

![joxi_screenshot_1539265326163](https://user-images.githubusercontent.com/13568351/46808330-c3391a80-cd74-11e8-9288-51ba7cba272e.png)
1. First popup button
2. Second popup button

**Comment by magento-engcom-team** [comment]
Hi @k1las. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@k1las do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by tgamanov** [comment]
I am also had this bug, only one WYSIWYG work properly.

**Comment by mmatrafailo** [comment]
@magento-engcom-team give me 2.2.3 instance

**Comment by magento-engcom-team** [comment]
Hi @k1las. Thank you for your request. I'm working on Magento 2.2.3 instance for you

**Comment by magento-engcom-team** [comment]
Hi @k1las, here is your [Magento instance](https://i-18534-2-2-3.engcom.dev.magento.com).
Admin access: [https://i-18534-2-2-3.engcom.dev.magento.com/admin](https://i-18534-2-2-3.engcom.dev.magento.com/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by mmatrafailo** [comment]
@magento-engcom-team Yes, I was able to reproduce the issue on the vanilla Magento instance.

**Comment by sidolov** [comment]
Hi @k1las. Thank you for your report. 
The issue has been fixed in magento/magento2#18535 by @k1las in 2.2-develop branch
Related commit(s):
 - [07ee1786beb48ea160d3a2b4f0f0617b59cfae5f](https://github.com/magento/magento2/commit/07ee1786beb48ea160d3a2b4f0f0617b59cfae5f)
 - [b5e915f4fd72928f7c30661f15616b9d1baba19f](https://github.com/magento/magento2/commit/b5e915f4fd72928f7c30661f15616b9d1baba19f)

The fix will be available with the upcoming 2.2.8 release.
